tests/app/views/test_brief_response.py

```diff
@@ -100,6 +129,47 @@ def test_create_new_brief_response(self, live_dos_framework, brief_response_data
         assert data['briefResponses']['supplierName'] == 'Supplier 0'
         assert data['briefResponses']['briefId'] == self.brief_id
 
+    def test_create_new_brief_response_with_only_one_page_question(self, live_dos_framework):
```

Comment:
I think instead the usage case will need to be that we create a brief response with no page questions (as in creating the brief response when you click the start application button rather than when you submit your first page of questions)